src/components/Popover/PopoverArrow.js

```diff
@@ -1,34 +1,34 @@
-import React from "react";
 import styled from "styled-components";
 import { prop } from "styled-tools";
 import as from "../../enhancers/as";
-import Perpendicular from "../Perpendicular";
 import Arrow from "../Arrow";
 import Box from "../Box";
 
-const PerpendicularArrowBox = Perpendicular.as([Arrow, Box]);
+const ArrowBox = Arrow.as(Box);
 
-const Component = props => (
-  <PerpendicularArrowBox
-    alignOffset={props.align !== "center" ? "0.5rem" : 0}
-    {...props}
-  />
-);
-
-const PopoverArrow = styled(Component)`
+const PopoverArrow = styled(ArrowBox)`
+  position: absolute;
   color: white;
   border: inherit;
   border-top: 0;
   font-size: 1.25em;
   border-radius: 0;
+  [data-placement^="top"] > & {
```

Comment:
what's the rationale behind the `&` here?

Reply:
That's a feature styled-components took from sass/less: https://css-tricks.com/the-sass-ampersand

Reply:
oh, the logic is to select a nested element with the same class, `&` ?

Reply:
It selects itself (when parent matches the selector `[data-placement^="top"]`)